tests/lib/tinyproxy/tinyproxy.py

```diff
@@ -111,9 +112,22 @@ def _read_write(self, soc, max_idling=20):
     do_PUT  = do_GET
     do_DELETE=do_GET
 
+
+def maybe_sd_notify(s: str) -> None:
```

Comment:
Do you think we should start having snapd.tests package where we can stash things like that?